faq-v1.md

```diff
@@ -48,6 +49,10 @@ We did not take this decision lightly, as many federal systems are operating at
 
 Importantly, we’re not just finalizing SP 800-63 and throwing it over the wall. We’re here to help agencies get it right all the way into production. As needs are identified, we will consider additional resources to support agency implementations. These resources will not contain additional normative requirements, but will instead provide further discussion, technology-specific examples, and details to support implementation of the guidelines. As agencies implement SP 800-63 for their specific use cases, we plan to be available to answer questions, provide clarifications regarding the guidelines, and apply any implementation-specific lessons learned to future revisions.
 
+<a name="q4"></a>**Q4: How do I know which xAL to choose?**
+
+There is always a trade off in choosing the appropriate level across all categories. Higher levels do come with greater security and assurance, but at the cost of greater complexity, overhead, and potential for failure. Therefore, it is not always a matter of choosing the highest possible level and then conforming to that. Instead, the risks and potential attacks against a given application need to be considered alongside the costs of implementing a given level. It is always possible for an implementation of an identity system to exceed the requirements of its levels in practice, and the assurance levels themselves have been designed to be subsumptive, such that fulfilling requirements for level 3 automatically fulfills (and in fact exceeds) the requirements for level 2 and level 1 of that category as well. The [core document of 800-63-3 section 6](https://pages.nist.gov/800-63-3/sp800-63-3.html#sec6) provides detailed discussion on choosing an appropriate combination of xAL's. 
```

Comment:
xAL's should be xALs (not possessive)
And while you're at it, my dictionary says trade-off is hyphenated.

Also, even with this FAQ, should we say something similar in the text (either errata or in next revision)? If so, we shouldn't close the issue. Don't know if you can change that now, maybe we should just reopen the issue after merge.

Reply:
OK, will update as indicated. The text referenced mostly already says what's stated here, but it's just not spelled out succinctly. It might be worth looking at section 6 again to see if we can make it stronger. I read through the text though and thought a FAQ entry would be a better place for this formulation.

This won't auto-close the issue in the other repository so we can address it there as well if we want to.